server/build.gradle

```diff
@@ -10,6 +10,7 @@ dependencies {
     implementation project(':conductor-contribs')
     implementation project(':conductor-redis-persistence')
     implementation project(':conductor-cassandra-persistence')
+    implementation project(':conductor-postgres-persistence')
```

Comment:
Can you also add `conductor-mysql-persistence`? That was also left out during the migration.